samples/bluetooth/fast_pair/locator_tag/src/fp_adv.c

```diff
@@ -493,32 +522,31 @@ uint8_t app_fp_adv_id_get(void)
 	return fp_adv_param.id;
 }
 
-int app_fp_adv_uninit(void)
+int app_fp_adv_init(void)
 {
 	int err;
 
-	app_fp_adv_mode_set(APP_FP_ADV_MODE_OFF);
-	app_fp_adv_rpa_rotation_suspend(false);
-
-	fp_adv_conn_clear();
-
-	err = fp_adv_set_teardown();
+	err = bt_fast_pair_fmdn_info_cb_register(&fmdn_info_cb);
```

Comment:
```suggestion
	size_t i = 0;
	static bool trigger_ctx[TRIGGER_NUM_MAX] /* Macro to define based end and start section pointers */

	STRUCT_SECTION_FOREACH(app_fp_adv_trigger, adv_trigger) {
		__ASSERT_NO_MSG(i < TRIGGER_NUM_MAX);
		adv_trigger->ctx = &trigger_ctx[i++];
	}

	err = bt_fast_pair_fmdn_info_cb_register(&fmdn_info_cb);
```

Reply:
If we want this mapping, then maybe we could use bitmask to reduce memory consumption? Otherwise, I would stick to @mkapala-nordic's approach to simplify code.